drivers/at24cxxx/at24cxxx.c

```diff
@@ -0,0 +1,257 @@
+/*
+ * Copyright (C) 2019 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_at24cxxx
+ * @{
+ *
+ * @file
+ * @brief       Device driver implementation for the AT24C128 / AT24C256 EEPROM units.
+ *
+ * @author      Fabian Hüßler <fabian.huessler@ovgu.de>
+ * @}
+ */
+
+#include <stdio.h>
+#include <errno.h>
+#include <string.h>
+
+#define ENABLE_DEBUG  (0)
+#include "debug.h"
+#include "assert.h"
+#include "xtimer.h"
+#include "byteorder.h"
+
+#include "include/at24cxxx_params.h"
+#include "include/at24cxxx_defines.h"
+#include "at24cxxx.h"
+
+#ifndef min
+#define min(a, b) ((a) > (b) ? (b) : (a))
+#endif
+
+static int32_t _read(const at24cxxx_t *dev, uint16_t pos, void *data,
+                     size_t len)
+{
+    int check;
+    uint8_t polls = AT24CXXX_MAX_POLLS;
+
+    while ((check =
+                i2c_read_regs(dev->params.i2c, dev->params.dev_addr, pos, data,
+                              len,
+                              I2C_REG16)) == -ENXIO) {
```

Comment:
(Just a comment, feel free to ignore.)

How about:

```C
    while (-ENXIO == (check = i2c_read_regs(dev->params.i2c,
                                            dev->params.dev_addr, pos, data,
                                            len, I2C_REG16))) {
```

This should be fine with RIOT's coding style and looks less fragmented to me.

Reply:
Before it gets merged, I will do a last uncrustify, then adjust the indentation like you suggest, then squash.